### PR TITLE
Added table formatting to output

### PR DESCRIPTION
**Pull request by nabadger**
Adds table formatting to output - see #125

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -5,12 +5,46 @@ import (
...
+	ColorHigh     = "\033[0;31m%s\033[0m"
+	ColorMedium   = "\033[0;33m%s\033[0m"
+	ColorLow      = "\033[0;94m%s\033[0m"
+	ColorDefault  = "\033[0;97m%s\033[0m"
+)
+
 var store = make(map[string][]*clair.Vulnerability)
 
+var Severity = map[string]int{
+	"Defcon1":    7,
```
Comment (hashmap) [inline main.go]: Why not to map severity directly to colors?

Comment (nabadger) [reply main.go]: Sure, will do

**Comment by jeff-knurek** [comment]
once I got Klar working, my first thought was to improve the output.

If you don't mind @nabadger, I'll send a PR to `nabadger:prettify_output` where I pull out the formatting from the main function, so that we can keep the default, and then expand/improve more formats in the future

**Comment by nabadger** [comment]
@jeff-knurek sounds good

**Comment by nabadger** [comment]
@hashmap this is good to review again. There's a nice change from @jeff-knurek which means this is no longer a breaking change (in terms of output), and it cleans up some of the code structure of how output is formatted.

Usage would now be something like:

```
FORMAT_OUTPUT=<standard|table|json> CLAIR_ADDR=localhost ./klar ...
```
but also supports existing usage:
```
JSON_OUTPUT=1 CLAIR_ADDR=localhost ./klar ...
```

Eventually we could deprecate `JSON_OUTPUT` in favour of `FORMAT_OUTPUT`

**Comment by hashmap** [comment]
Great work, thanks!

**Pull request merged**